sbnanaobj/StandardRecord/SRSlice.h

```diff
@@ -42,8 +43,10 @@ namespace caf
       SRFlashMatch fmatch_a; //!< Optical flash-match for this slice of TPC charge
       SRFlashMatch fmatch_b; //!< Optical flash-match for this slice of TPC charge
 
-      SRFakeReco fake_reco;
+      SROpT0Finder opt0;       //!< OpT0Finder (flash-match and Q->L); filled with the **highest OpT0 score** assoc. to the slice 
+      SROpT0Finder opt0_sec;   //!< Secondary OpT0Finder (flash-match and Q->L); filled with the **second highest OpT0 score**  assoc. to the slice 
```

Comment:
I immediately read this variable as meaning "seconds not nanoseconds". It's a foolish assumption especially at an object level but I can imagine that I won't be the only one who makes it when just scrolling through a CAF in TBrowser.

How about `opt0_2nd` (thanks @afilkins) ?

Reply:
I'd also be happy with `opt0_2nd` (I did have to ask other people for ideas about this naming scheme too lol) 

If it turns out the `fmatch_a` and `fmatch_b` get dropped by Rob though, I might consider taking over that structure (`opt0` as default and `opt0_a` and `opt0_b` for primary/secondary if people need both). Hopefully that wouldn't be confusing?? It would mean that `opt0_a` is always a duplicate of `opt0`, I'm not sure if that's unwanted behavior.

Reply:
Yeah that sounds like a reasonable approach to me! I would say we don't want to add identical duplicates. I wonder whether we can make two names point at the same object? @PetrilloAtWork might know about this?

Another option would be to make a vector? Then the size would normally be 1 and you could make the hierarchy of primary, secondary etc just be natural in the order of the vector?

Reply:
I think that making a vector of OpT0 objects (similar to how there's a vector of showers, I think?) is a structure that could be troublesome for analyzers who use Python/uproot (such as myself), so I'm trying to avoid adding new vectors to the cafs 😛 

If we can figure out how to get the two names pointing to the same object working, I think that'd a good approach